.swift-version

```diff
@@ -1 +1 @@
-3.0
+3.2
```

Comment:
Pretty sure this isn't necessary, as it's only used by CocoaPods to run the local lint build when doing a `pod trunk push`. Since Xcode only supports `3.0` or `4.0` (in Xcode 9) for this setting, only `3.0` should be necessary.